sotu.md

```diff
@@ -190,6 +190,7 @@ of the Haskell runtime.
 
 * [`warp`](https://hackage.haskell.org/package/warp) / [`wai`](https://hackage.haskell.org/package/wai) - the low-level server and API that all server libraries share, with the exception of `snap`
 * [`scotty`](https://hackage.haskell.org/package/scotty) - A beginner-friendly server framework analogous to Ruby's Sinatra
+* [`spock`](https://www.spock.li/) - Lighter than the "enterprise" frameworks, but more featureful than spock (type-safe routing, sessions, conn pooling, csrf protection, authentication, etc)
```

Comment:
This doesn't read correctly. Is it supposed to say "more featureful than scotty"?